app/src/main/java/grinnell/appdev/edu/lyles/fragments/MenuTabFragment.java

```diff
@@ -0,0 +1,106 @@
+package grinnell.appdev.edu.lyles.fragments;
+
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.ExecutionException;
+
+import grinnell.appdev.edu.lyles.AsyncRetrieval;
+import grinnell.appdev.edu.lyles.MenuTabListAdapter;
+import grinnell.appdev.edu.lyles.R;
+
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.DRINKS_INDEX;
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.HOT_FOOD_INDEX;
+import static grinnell.appdev.edu.lyles.MenuViewPagerAdapter.SNACKS_INDEX;
+
+/**
+ * A fragment to display available drinks to users
+ */
+public class MenuTabFragment extends Fragment {
+
+    private final int mPosition;
+
+    public MenuTabFragment() {
+        this.mPosition = -1;
+    }
+
+    public MenuTabFragment(int position) {
+        this.mPosition = position;
+    }
+
+    private String retrieveJson() {
+        try {
+            String url;
+            switch (this.mPosition) {
+                case HOT_FOOD_INDEX:
+                    url = "http://www.cs.grinnell.edu/~birnbaum/appdev/lyles/hotfood.json";
```

Comment:
I figured these URLs would be temporary, since we have only been using them for testing purposes, and presumably the production URL would be somewhere more permanent than Mattori's MathLAN account.